Makefile

```diff
@@ -15,10 +15,18 @@ define cli_envs
 	-e BINARIS_LOG_ENDPOINT
 endef
 
+BRANCH := $(shell if [[ ! -z $${BRANCH_NAME+x} ]]; then echo $${BRANCH_NAME}; else git rev-parse --abbrev-ref HEAD 2>/dev/null || echo UNKNOWN; fi)
+
 .PHONY: build
 build: require-tag
 		$(DOCKER) build $(NO_CACHE) -f binaris.Dockerfile -t $(DOCKER_IMAGE):$(tag) .
 
+.PHONY: tag
+tag: require-tag
+		$(DOCKER) tag $(DOCKER_IMAGE):$(tag) binaris
+		$(DOCKER) tag $(DOCKER_IMAGE):$(tag) $(DOCKER_IMAGE):latest
```

Comment:
we should probably not do `latest` at all. nothing good comes out of it.

Reply:
https://github.com/binaris/binaris/issues/270